addons/binding/org.openhab.binding.loxone/src/main/java/org/openhab/binding/loxone/internal/core/LxWsClient.java

```diff
@@ -258,16 +262,33 @@ boolean connect() {
             socket = new LxWebSocket();
             wsClient = new WebSocketClient();
 
+            String message = socket.httpGet(CMD_CFG_API);
+            if (message != null) {
+                LxJsonSubResponse response = socket.getSubResponse(message);
+                if (response != null && response.code == 200 && response.value != null) {
+                    try {
+                        LxJsonCfgApi cfgApi = gson.fromJson(response.value.getAsString(), LxJsonCfgApi.class);
+                        swVersion = cfgApi.version;
+                        macAddress = cfgApi.snr;
+                    } catch (JsonSyntaxException | NumberFormatException e) {
+                        logger.debug("[{}] Error parsing API config response: {}, {}", debugId, response,
+                                e.getMessage());
+                    }
+                }
```

Comment:
Possible log in the else? For example when response code != 200? Or propagate to the user with state change? (this might apply for other else states that only log)

Reply:
Added debug log and also found that I don't account for null software version later, fixed.